controller/network/network.go

```diff
@@ -587,6 +587,10 @@ func (network *Network) GetServiceCache() Cache {
 	return network.Services
 }
 
```

Comment:
Shouldn't need this. RouterController is already in Handlers for access by edge functionality.

Reply:
Ah I see, not `GetRouterController` but `appEnv.Handlers.Router.Routers`